data/intro missions.txt

```diff
@@ -63,7 +63,15 @@ mission "Intro [0]"
 			label end
 			`	"Great," he says. "My name is James, by the way."`
 			`	"<first> <last>," you say.`
+			branch MZ
+				or
+					has "name: Michael Zahniser"
+					has "name: M Z"
 			`	"I'm looking forward to traveling with you, Captain <last>," says James.`
+				goto MZend
+			label MZ
+			`	"<first> <last>?" he stops for a moment. "I've heard that name before. Reminds me of someone famous, I think," says James. "Anyways, I'm looking forward to traveling with you."`
+			label MZend
```

Comment:
I think just the full name should be accepted. MZ's name appears in the credits, so it isn't a stretch for someone to try out his full name, but "M Z" seems like something someone would only choose because they know of this easter egg, as in the community we just type "MZ" without the space, so having your first name be M and last name by Z seems like likely to be something someone just does.
```suggestion
			branch MZ
				has "name: Michael Zahniser"
			`	"I'm looking forward to traveling with you, Captain <last>," says James.`
				goto end
			label MZ
			`	"<first> <last>?" he stops for a moment. "I've heard that name before. Reminds me of someone famous, I think," says James. "Anyways, I'm looking forward to traveling with you."`
			label end
```